src/main/java/com/virtudoc/web/controller/CurrentAppointmentsController.java

```diff
@@ -18,40 +18,41 @@
     @GetMapping("/notifications")
     String getNotifications(Model model) {
         //Manually add 3 appointments
-//        Appointment apt1 = new Appointment();
-//        apt1.setEmail("test@test.com");
-//        apt1.setPatientName("Patient 1");
-//        apt1.setSymptoms("Covid");
-//        apt1.setDoctorName("Doctor 1");
-//        apt1.setLocation("Kennesaw");
-//        apt1.setDate("Feb 3");
-//        apt1.setReasonForVisit("Reason for Visit");
-//
-//        Appointment apt2 = new Appointment();
-//        apt2.setEmail("test@test.com");
-//        apt2.setPatientName("Patient 1");
-//        apt2.setSymptoms("Covid");
-//        apt2.setDoctorName("Doctor 1");
-//        apt2.setLocation("Kennesaw");
-//        apt2.setDate("Feb 3");
-//        apt2.setReasonForVisit("Reason for Visit");
-//
-//        Appointment apt3 = new Appointment();
-//        apt3.setEmail("test@test.com");
-//        apt3.setPatientName("Patient 1");
-//        apt3.setSymptoms("Covid");
-//        apt3.setDoctorName("Doctor 1");
-//        apt3.setLocation("Kennesaw");
-//        apt3.setDate("Feb 3");
-//        apt3.setReasonForVisit("Reason for Visit");
-//
-//        model.addAttribute("notifications", Arrays.asList(
-//                apt1, apt2, apt3
-//        ));
+        Appointment apt1 = new Appointment();
+        apt1.setEmail("test@test.com");
+        apt1.setPatientName("John Smith");
+        apt1.setSymptoms("Covid");
+        apt1.setDoctorName("Dr. Mary Jane");
+        apt1.setLocation("Kennesaw Side Building");
+        apt1.setDate("2PM Feb 3");
+        apt1.setReasonForVisit("Covid-19 Test");
+
+        Appointment apt2 = new Appointment();
+        apt2.setEmail("test@test.com");
+        apt2.setPatientName("Jane Smith");
+        apt2.setSymptoms("Covid");
+        apt2.setDoctorName("Dr. Phil Johnson");
+        apt2.setLocation("Kennesaw Main Building");
+        apt2.setDate("3PM Feb 4");
+        apt2.setReasonForVisit("Covid-19 Test");
+
+        Appointment apt3 = new Appointment();
+        apt3.setEmail("test@test.com");
+        apt3.setPatientName("John Smith");
+        apt3.setSymptoms("Covid");
+        apt3.setDoctorName("Dr. Mary Jane");
+        apt3.setLocation("Marietta Main Building");
+        apt3.setDate("1PM Feb 7");
+        apt3.setReasonForVisit("Annual Exam");
+
+        model.addAttribute("appointments", Arrays.asList(
+                apt1, apt2, apt3
+        ));
+
 
         //Pull appointments from DB
-        List<Appointment> allAppointments = service.listAll();
-        model.addAttribute("notifications", allAppointments);
+//        List<Appointment> allAppointments = service.listAll();
+//        model.addAttribute("appointments", allAppointments);
```

Comment:
Not really relevant for this PR, but in the future if you want there is now a database seeder located at `src/main/java/com/virtudoc/web/runner/DatabaseSeedRunner.java` if you want to use that instead.